src/glyphs.js

```diff
@@ -90,12 +93,33 @@ const processMessageRequest = (req) => {
   }
 
   if (!req.body.pass || sha256(req.body.pass + "719GxFYNgo") !== "700e78f75bf9abb38e9b2f61b227afe94c204947eb0227174c48f55a4dcc8139") {
-    processed.errors.push('Invalid password.')
+    processed.errors.push('Invalid password.');
   }
 
   return processed;
 };
 
+const processImportRequest = (req) => {
+  const processed = {
+    errors: []
+  };
+
+  if (!req.files[""]) {
+    processed.errors.push('No file provided!');
+  }
+
+  if (req.fields.type !== "append" && req.fields.type !== "overwrite") {
+    processed.errors.push('Invalid import type.');
+  }
+
+  if (!req.fields.pass || sha256(req.fields.pass + "719GxFYNgo") !== "700e78f75bf9abb38e9b2f61b227afe94c204947eb0227174c48f55a4dcc8139") {
```

Comment:
Do we have these salt constants copied and pasted in multiple places now?  If so, we might want to move them to their own constant at the top of the file or so.